<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuchungsFenster</class>
 <widget class="QWidget" name="BuchungsFenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>35</y>
     <width>331</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditID"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Von</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dateEditVon">
          <property name="displayFormat">
           <string>dd/MMMM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Bis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dateEdit_2">
          <property name="displayFormat">
           <string>dd/MMMM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Preis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Zug">
       <attribute name="title">
        <string>Zug</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>12</y>
          <width>291</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="Von_2">
           <property name="text">
            <string>Von</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QLineEdit" name="lineEditAbfahrtZ"/>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Abfahrt</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="2">
          <widget class="QTimeEdit" name="timeEditAbfahrt">
           <property name="maximumTime">
            <time>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string>hh:mm AP</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="Bis_2">
           <property name="text">
            <string>Nach</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="3">
          <widget class="QLineEdit" name="lineEditAnkunftZ"/>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Ankunft</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="2">
          <widget class="QTimeEdit" name="timeEditAnkunft"/>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Zugklasse</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLineEdit" name="lineEditZugklasse"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Ãœber</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="3">
          <widget class="QTableWidget" name="tableWidgetZ">
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Flug">
       <attribute name="title">
        <string>Flug</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="Von_3">
          <property name="text">
           <string>From Dest</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditAbfahrtF"/>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>from Dest(N)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditFDestcode">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Bis_3">
          <property name="text">
           <string>To Dest</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditAnkunftF"/>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>To Dest(N)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdittoDestCode">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>KLasse</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditFlugKlasse"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Airline</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditAirline"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Hotel">
       <attribute name="title">
        <string>Hotel</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="Von_4">
          <property name="text">
           <string>Hotel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditHotel"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Bis_4">
          <property name="text">
           <string>Town</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditTown"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Zimmertyp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditZimmertyp"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Car">
       <attribute name="title">
        <string>Car</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="Von_5">
          <property name="text">
           <string>Pickup Loc</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditPickup"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Bis_5">
          <property name="text">
           <string>Return Loc</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditReturn"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Cartyp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditCartyp"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Company</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditComany"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
